config/management/operational/src/command.rs

```diff
@@ -90,9 +100,11 @@ impl std::fmt::Display for CommandName {
 impl Command {
     pub fn execute(self) -> String {
         match self {
-            Command::InsertWaypoint(cmd) => {
-                format!("{:?}", cmd.execute().map(|()| "success").unwrap())
-            }
+            Command::AddValidator(cmd) => format!("{:?}", cmd.execute().unwrap()),
+            Command::InsertWaypoint(cmd) => cmd.execute().map(|()| "success".into()).unwrap(),
+            Command::ExtractPrivateKey(cmd) => cmd.execute().map(|()| "success".into()).unwrap(),
```

Comment:
If we're extracting PrivateKeys, do we force HTTPS on the json_endpoint?

Reply:
Great question, I hope no one uses this and it is blocked by the tokens that folks use. Maybe we could add an extra parameter --i-know-what-i-am-doing :)